scripts/Services/work-items.service.spec.ts

```diff
@@ -160,38 +129,36 @@ describe("WorkItemsService", function() {
         });
 
         beforeEach(function() {
-            getInProgressWorkItemRefsDeffered = new Deferred<WorkItemReference[]>();
-            getWorkItemsDeffered = new Deferred < WorkItem[] > ();
-            getWorkItemRefsByWIQLDeffered = new Deferred< WorkItemReference[] > ();
-
-            getWIs = sandbox.stub(workItemTrackingClient, 'getWorkItems').returns(getWorkItemsDeffered.promise);
-            getWisRefsClientStub = sandbox.stub(workItemTrackingClient, 'getInProgressWorkItemRefs').returns(getInProgressWorkItemRefsDeffered.promise);
             witService = new WorkItemsService(workItemTrackingClient);
+
+            getWorkItemsStub = sandbox.stub(workItemTrackingClient, 'getWorkItems').returns(Promise.resolve(responseWIs));
+            getInProgressWorkItemRefsStub = sandbox.stub(workItemTrackingClient, 'getInProgressWorkItemRefs').returns(Promise.resolve(responseRefs));
         });
 
         afterEach(function() {
             sandbox.restore();
         });
 
-        it("Should call wit client", function() {
-            witService.getInProgressWorkItems();
```

Comment:
I think this is where the error was coming from, as the code uses promises but the callback doesn't use `done`.